test/e2e/tests/swap-eth.spec.js

```diff
@@ -0,0 +1,80 @@
+const { strict: assert } = require('assert');
+const { withFixtures, regularDelayMs } = require('../helpers');
+
```

Comment:
regularDelayMs doesn't seem to be used

Reply:
You can run `yarn lint` to identify any issues

Reply:
updated